deployments/terraform/modules/gateway/alarms.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_cloudwatch_metric_alarm" "success-rate" {
+  alarm_name          = "${var.family}-${var.application}-success-rate"
+  count               = var.alarm_create ? 1 : 0
+  datapoints_to_alarm = var.datapoints_to_alarm
+  comparison_operator = "LessThanThreshold"
+  evaluation_periods  = var.evaluation_periods
+  threshold           = var.success_rate_threshold
+  treat_missing_data  = "notBreaching"
+
+  metric_query {
+    id          = "requests"
+    return_data = false
+
+    metric {
+      dimensions = {
+        "path" = var.api_request_path
+      }
+      metric_name = "ApiRequestCount"
```

Comment:
i tend to go with the request count from the load balancer, as we would list all paths from a gateway here as we have quite some gateways with multiple paths.